heuristic/classes/Solution.py

```diff
@@ -165,3 +165,38 @@ def to_file(self, location: str):
                     print(f"V{idx_route},{leg},S{idx_stack},{stack}", file=file)
 
         file.close()
+
+    def to_graph(self):
```

Comment:
If you can, I'd add this as a top plot to the existing plotting facilities. You might want to use [this](https://stackoverflow.com/questions/21160914/python-matplotlib-creating-two-subplots-with-different-sizes) as a building block (matplotlib's [GridSpec](https://matplotlib.org/tutorials/intermediate/gridspec.html) is probably the tool to use).